pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -675,3 +684,46 @@ type DigitalOceanSDConfig struct {
 	// +optional
 	RefreshInterval *v1.Duration `json:"refreshInterval,omitempty"`
 }
+
+// Docker SD configurations allow retrieving scrape targets from Docker Engine hosts.
+// This SD discovers "containers" and will create a target for each network IP and port the container is configured to expose.
+type DockerSDConfig struct {
+	// Address of the docker daemon
```

Comment:
```suggestion
	// Address of the docker daemon
	// +kubebuilder:validation:MinLength=1
```

Reply:
Added the relevant kubebuilder comment for code-gen. Resolving this comment.